beacon-chain/rpc/beacon/attestations_test.go

```diff
@@ -612,6 +612,12 @@ func TestServer_ListIndexedAttestations_NewStateManagnmentDisabled(t *testing.T)
 func TestServer_ListIndexedAttestations_GenesisEpoch(t *testing.T) {
 	params.OverrideBeaconConfig(params.MainnetConfig())
 	defer params.OverrideBeaconConfig(params.MinimalSpecConfig())
+
+	config := &featureconfig.Flags{
+		NewStateMgmt: true,
+	}
+	featureconfig.Init(config)
+
```

Comment:
Just to double check: am I right in assuming that when `featureconfig.Init()` is called, it amends the global state, so changes are persistent, and can affect other tests? If so, then it would probably be better to reset config back (the same way we tear down the database)?

In that assumption, whenever global state change is required I did the following:
https://github.com/prysmaticlabs/prysm/blob/0806807d613e7241afc8a5d5b774e415ac2663bc/beacon-chain/rpc/beacon/attestations_test.go#L615-L616

where 
https://github.com/prysmaticlabs/prysm/blob/0806807d613e7241afc8a5d5b774e415ac2663bc/beacon-chain/rpc/beacon/attestations_test.go#L1236-L1245

Please let me know if that assumption is incorrect, or you think that resetting global state is an overkill.

Reply:
Good point. I addressed it for all the instances